src/mmrelay/meshtastic/connection.py

```
@@ -57,6 +57,46 @@ class ConnectionCompletedWithoutClientError(ConnectionError):
+
+
+def _optional_exception_tuple(*candidates: object) -> tuple[type[BaseException], ...]:
+    exception_types = tuple(
+        candidate
+        for candidate in candidates
+        if isinstance(candidate, type) and issubclass(candidate, BaseException)
+    )
+    return exception_types or (_NoTypedBleError,)
+
+
+def _modern_ble_library_owns_stale_cleanup(supports_auto_reconnect: bool) -> bool:
```

Ruff (0.15.11)

[warning] Boolean-typed positional argument in function definition

(FBT001)